src/renderer/store/mulmo_script_history.ts

```
@@ -92,8 +92,12 @@
     return null;
   });
   const isValidScript = computed(() => {
-    return zodError.value.success;
+    const beatIds = new Set((currentMulmoScript.value?.beats ?? []).map(beat => beat.id));
```

ESLint

[error] Replace `beat` with `(beat)`

(prettier/prettier)